docs/content/docs/libs/state_processor_api.md

```diff
@@ -515,6 +515,28 @@ Before you interrogate state using the table API, make sure to review our [Flink
 
 IMPORTANT NOTE: State Table API only supports keyed state.
 
+### Metadata
+
+The SQL connector allows users to read metadata of savepoints in the following way:
+```SQL
+LOAD MODULE state;
```

Comment:
nit: if this is savepoints only ; should we call it savepoint-state. Currently the implementation seems to be tied to a fixed savepoint schema. savepoint-state would be more descriptive and better scoped - allowing us to provide a more generic state module later - if we want to.

Reply:
Seems like all the rest of the questions related `is savepoint only?`. My answer to all of them is no. We've agreed on this naming in the discussion.